Makefile

```diff
@@ -0,0 +1,10 @@
+# This Makefile is used to compile an LND binary for the integration tests.
+
+GO_BUILD := go build
+CARGO_TEST := cargo test
+LND_PKG := github.com/lightningnetwork/lnd
+
+itest:
+	@$(call print, "Building lnd for itests.")
+	$(GO_BUILD) -tags="peersrpc signrpc dev" -o /tmp/lndk-tests/bin/lnd-itest$(EXEC_SUFFIX) $(LND_PKG)/cmd/lnd
+	$(CARGO_TEST) -- --test-threads=1 --nocapture
```

Comment:
Am I correct in thinking that this will also run the unit tests? That doesn't really make sense to me. 

Can we use `cargo test --test {itest module}` to only run the itests? Likewise, we should have a way to only run unit tests.

Reply:
Good point, yes I'll update this

Reply:
I wrote up some instructions in `CONTRIBUTING` to specify how to run integration tests vs. unit tests